config/initializers/content_security_policy.rb

```diff
@@ -1,9 +1,3 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
Worth leaving this note in place - might help future developers confused by their changes are having no impact - below is the rails 7.1 content which should be here

```
# Be sure to restart your server when you modify this file.

# Define an application-wide content security policy.
# See the Securing Rails Applications Guide for more information:
# https://guides.rubyonrails.org/security.html#content-security-policy-header
```